manifests/init.pp

```diff
@@ -24,6 +24,8 @@
   String[1]               $service_name       = 'keepalived',
   Optional[String[1]]     $service_restart    = undef,
 
+  Boolean                 $package_manage     = true,
```

Comment:
It looks like most modules with such a feature prefer to use `manage_package` rather than `package_manage`:

```sh-session
romain@zappy ~/Projects/voxpupuli/modulesync_config/modules/voxpupuli % grep manage_package */manifests/init.pp | wc -l
      43
romain@zappy ~/Projects/voxpupuli/modulesync_config/modules/voxpupuli % grep package_manage */manifests/init.pp | wc -l
       1
```

Do you think we should match the other modules for consistency?

`manage_service` vs. `service_manage`shows similar (but less extreme) usage:

```sh-session
romain@zappy ~/Projects/voxpupuli/modulesync_config/modules/voxpupuli % grep manage_service */manifests/init.pp | wc -l
      58
romain@zappy ~/Projects/voxpupuli/modulesync_config/modules/voxpupuli % grep service_manage */manifests/init.pp | wc -l
      22
```

Reply:
I don't care. I'm just interested by the functionality. If `manage_package` is more popular, I can change.

Reply:
hey. Since most of our modules use `manage_package` I think it's good if you update to that name.
Cheers, Tim

Reply:
`package_manage` is replaced by `manage_package`